front/app/containers/Admin/projects/routes.tsx

```diff
@@ -293,19 +293,19 @@ const createAdminProjectsRoutes = () => {
                   </PageLoading>
                 ),
               },
-              {
-                path: projectsRoutes.projectNewPhase,
-                element: (
-                  <PageLoading>
-                    {/* We use the key here to make sure that the component is treated as a different instance
-                    to differentiate between the new and edit phase. This distinction is especially important
-                    when the component is already visible and the route changes to the same component.
-                    For example, from phase setup to creating a new phase.
-                    */}
-                    <AdminPhaseNewAndEdit key="new" />
-                  </PageLoading>
-                ),
-              },
+              // {
+              //   path: projectsRoutes.projectNewPhase,
+              //   element: (
+              //     <PageLoading>
+              //       {/* We use the key here to make sure that the component is treated as a different instance
+              //       to differentiate between the new and edit phase. This distinction is especially important
+              //       when the component is already visible and the route changes to the same component.
+              //       For example, from phase setup to creating a new phase.
+              //       */}
+              //       <AdminPhaseNewAndEdit key="new" />
+              //     </PageLoading>
+              //   ),
+              // },
```

Comment:
Commented out to show this route definition does nothing. It's the same as the :phaseId path below it, since the same components is rendered and the key prop is not doing anything. `new` is simply interpreted as the `phaseId`.

I wonder how robust our route types solution is (or what it actually does) if none of these mistakes I point out here have come to the surface. 🤔